zhaquirks/__init__.py

```diff
@@ -131,14 +134,21 @@ def _calculate_battery_percentage(self, raw_value):
         if raw_value in (0, 255):
             return -1
         volts = raw_value / 10
-        if volts < self.MIN_VOLTS:
-            volts = self.MIN_VOLTS
-        elif volts > self.MAX_VOLTS:
-            volts = self.MAX_VOLTS
+        volts = max(volts, self.MIN_VOLTS)
+        volts = min(volts, self.MAX_VOLTS)
+
+        percent = round(
+            ((volts - self.MIN_VOLTS) / (self.MAX_VOLTS - self.MIN_VOLTS)) * 200
+        )
 
-        percent = ((volts - self.MIN_VOLTS) / (self.MAX_VOLTS - self.MIN_VOLTS)) * 200
+        log_msg = (
+            f"{self.endpoint.device.manufacturer} {self.endpoint.device.model}, "
+            f"Voltage [RAW]:{raw_value} [Max]:{self.MAX_VOLTS} "
+            f"[Min]:{self.MIN_VOLTS}, Battery Percent: {percent/2}"
```

Comment:
we shouldn't use F-Strings for logging, ie no need to format something that might not see the light of the day at all.

Reply:
@roblandry wanna change this so it can be merged?

Reply:
Done